install_pim/docker/installation_docker.rst

```diff
@@ -73,7 +73,7 @@ You need to get a PIM Enterprise Standard archive from the Partners Portal. See
     $ mkdir pim
     $ tar -xvjf pim-enterprise-standard-v4.0.tar.gz -C pim
     $ cd pim
-    $ docker run -u www-data -v $(pwd):/srv/pim -w /srv/pim --rm akeneo/pim-php-dev:4.0 \
+    $ docker run -u www-data -v $(pwd):/srv/pim -v ~/.composer:/var/www/.composer -w /srv/pim --rm akeneo/pim-php-dev:4.0 \
```

Comment:
is it OK? or should I put `$HOST_COMPOSER_HOME` instead of `~/.composer`?

Reply:
HOST_COMPOSER_HOME is confusing and only needed for people who decided to change the default composer folder, which is a very rare situation.